samza-core/src/test/java/org/apache/samza/processor/TestStreamProcessor.java

```diff
@@ -115,19 +130,14 @@ public void onFailure(Throwable t) {
 
```

Comment:
Similar to below, it is worth asserting that failure is not called on a clean shutdown (though I'm less worried about this case).